UI/Utils/Mapper.cs

```diff
@@ -0,0 +1,48 @@
+﻿using Domain.Models;
+using Infrastructure.DTOs;
+
+namespace UI.Utils;
+
+public class Mapper
+{
+    public List<PlayerRequest> MapPlayersToPlayerRequests(List<Player> players)
+    {
+        var playerRequests = new List<PlayerRequest>();
+        foreach (var player in players)
+        {
+            var playerRequest = new PlayerRequest
+            {
+                Name = player.Name,
+                Password = player.Password,
+                Email = player.Email
+            };
+            playerRequests.Add(playerRequest);
+        }
+
+        return playerRequests;
+    }
+    
+    public PlayerRequest MapPlayerToPlayerRequest(Player player)
+    {
+        return new PlayerRequest
+        {
+            Name = player.Name,
+            Password = player.Password,
+            Email = player.Email
+        };
+    }
+    
+    public Player MapPlayerRequestToPlayer(PlayerRequest playerRequest)
+    {
+        return new Player
+        {
+            Name = playerRequest.Name,
+            Password = playerRequest.Password,
+            Email = playerRequest.Email,
+            CreatedAt = DateTime.UtcNow,
+            UpdatedAt = DateTime.UtcNow,
```

Comment:
Object PlayerRequest comes from Update endpoint? Or from Create as well? If so, leave UpdatedAt field empty, a player just created, and never updated, make no sense to have newly created played with UpdatedAt field set.

Reply:
fixed

Reply:
fixed